cime_config/usermods_dirs/NEON/FATES/defaults/user_nl_clm

```diff
@@ -0,0 +1,28 @@
+!----------------------------------------------------------------------------------
```

Comment:
In above I suggest that this would look like an include_user_mods file that looks like

../../_includes/defaults

Then in cime_config/usermods_dirs/NEON/FATES/defaults you'd just have the FATES specific changes which I think is just the history variable list and the surface dataset filename

In the big-leaf defaults it would similarly have a include_user_mods directory that just points to the baseclass version and then includes the regular surface dataset and the history fields list for big-leaf.

Reply:
Leave for now but know that we are going to update this in the future.